src/api/requests/Websocket/Websocket.ts

```diff
@@ -1,45 +1,50 @@
-import {Centrifuge, MessageContext} from "centrifuge";
-import {BaseWebsocketModel, WebsocketTransaction} from '@/api/types/transaction'
+import { Centrifuge, MessageContext } from 'centrifuge'
+import { BaseWebsocketModel, WebsocketTransaction } from '@/api/types/transaction'
+import { toast } from 'react-toastify'
 
 export const SetupWebsocket = (url: string) => {
```

Comment:
we don't return anything that could allow us, to disconnect from this websocket for example on logging out or something. I don't know if this would really be a problem, but worth checking

Reply:
If we will need disconnect or similar we should convert this function into hook